packages/vue/src/Charts/BarChart/OcBarChart.vue

```diff
@@ -77,13 +77,8 @@ const options = computed(() => ({
 
       let value = params.value;
 
-      if (props.tooltipCurrency) {
-        const currency = new Intl.NumberFormat("en-US", {
-          style: "currency",
-          currency: props.tooltipCurrency,
-        });
-
-        value = currency.format(params.value);
+      if (props.tooltipValueFormatter) {
+        value = props.tooltipValueFormatter(value);
       }
```

Comment:
```suggestion
      if (typeof props.tooltipValueFormatter === 'function') {
        value = props.tooltipValueFormatter(value);
      }
      
      if (props.tooltipCurrency) {
        const currency = new Intl.NumberFormat("en-US", {
          style: "currency",
          currency: props.tooltipCurrency,
        });
        value = currency.format(params.value);
      }
```

Reply:
@HoaiLee Yes, we can do it but I think for this we need only use `tooltipValueFormatter`, because for displaying currencies in hitpay we have check for zeroDecimalCurrencies. And I think more correct use `tooltipValueFormatter` for formatting tooltip